php-practice_1.php

```diff
@@ -1,41 +1,218 @@
 <?php
 // Q1 変数と文字列
+$name = '秋田';
 
 
+$message = '私の名前は「'.$name.'」です。';
```

Comment:
演算子の左右は読みやすさのため半角スペースを開けることが推奨されています。
```suggestion
$message = '私の名前は「' . $name . '」です。';
```